<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAddTransformer</class>
 <widget class="QMainWindow" name="frmAddTransformer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Transformer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtPro">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDatabase">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSub">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFed">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Database:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Substation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Feeder:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtProNum">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Project Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMinTrn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Transformer Size in kVA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaxTrn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum Transformer Size in kVA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBuffDist">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tranformer Buffer in Meter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdOK">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Resources/FormIcons/resources.qrc">
      <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/formaccept.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/formaccept.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdHelp">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Resources/FormIcons/resources.qrc">
      <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basichelp.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basichelp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdClose">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Resources/FormIcons/resources.qrc">
      <normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basicexit.png</normaloff>:/plugins/ElectricSystems/Resources/FormIcons/basicexit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>62</width>
      <height>62</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/plugins/ElectricSystems/Resources/FormIcons/basicnreca.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Dataset</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Project Parameter</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>txtPro</zorder>
   <zorder>txtDatabase</zorder>
   <zorder>txtSub</zorder>
   <zorder>txtFed</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>txtProNum</zorder>
   <zorder>label_5</zorder>
   <zorder>txtMinTrn</zorder>
   <zorder>label_6</zorder>
   <zorder>txtMaxTrn</zorder>
   <zorder>label_7</zorder>
   <zorder>txtBuffDist</zorder>
   <zorder>label_8</zorder>
   <zorder>cmdOK</zorder>
   <zorder>cmdHelp</zorder>
   <zorder>cmdClose</zorder>
   <zorder>frame_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>378</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Resources/FormIcons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
